typeCast.js

```diff
@@ -5,7 +5,7 @@ export {
 };
 
 
-const stringFromArrayBuffer = function (arrayBuffer, encoding = "utf-8") {
+const stringFromArrayBuffer = function (arrayBuffer, encoding = `utf-8`) {
```

Comment:
The `` instead of "" seems really weird.

Reply:
I understand, it takes some time to get used to it